showcase/src/assets/examples/date-range-picker/full.html

```diff
@@ -1,16 +1,16 @@
 <stark-date-range-picker
-		[dateFilter]="customDateFilter"
 		rangePickerId="id-range-picker"
 		rangePickerName="name-range-picker"
-		[endDate]="endDate"
-		endDateLabel="To"
-		[endMinDate]="minDate"
-		[endMaxDate]="maxDate"
 		[isDisabled]="isDisabled"
+		[dateFilter]="customDateFilter"
 		[startDate]="startDate"
-		startDateLabel="From"
+		startDateLabel="STARK.DATE_RANGE_PICKER.FROM"
```

Comment:
To improve the readability of the examples, we put the English translation in the other examples. 
 Could you please change this for `startDateLabel="From"` ?